README.md

```diff
@@ -2,15 +2,59 @@
 
 This package was originally developed to help release [Embroider](https://github.com/embroider-build/embroider) and was extracted so everyone can use it 🎉
```

Comment:
This probably isn't necessarry as part of the intro readme, especially if it's ready for other projects to use.

Reply:
true -- I'll move to the bottom